department-of-veterans-affairs/va.gov-cms
# VACMS 2681: Alter Redirect view to work around redirect_options behavior.

## Description

See #2681. 

This PR edits the displays of the Redirects view to display the redirect destination, as one would expect, rather than the "Server" or "Template" string.  I was not able to replicate the issue reported wherein edited redirects no longer worked.

## Testing done
Edited a few redirects, essentially at random, and verified that they continued to function and that the view displayed a linked destination path rather than a "Server" or "Template" string.  Also verified that edited redirects continued to function, even after clearing caches, etc.

## Screenshots
<img width="1742" alt="Screen Shot 2021-01-11 at 3 01 52 PM" src="https://user-images.githubusercontent.com/1318579/104232207-02f08000-541e-11eb-981c-41e5b0d08009.png">

## QA steps
As a user with admin access:
1. View the [admin redirects page](http://va-gov-cms.lndo.site/admin/config/search/redirect)
2. Then validate Acceptance Criteria from issue #2681:
    - [x] When editing a redirect,and saving it should not say "Server" in the destination. It should show the path like it did before editing.
    - [x] An edited redirect should actually redirect.

As a user with the 'Redirect Admin' role:
1. View the [non-admin redirects list](http://va-gov-cms.lndo.site/redirects-list)
2. Then validate Acceptance Criteria from issue #2681:
    - [x] When editing a redirect,and saving it should not say "Server" in the destination. It should show the path like it did before editing.

### Definition of Done


### Select Team for PR review

### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
